### ISSUE TITLE
ETH <-> DOGE conversion is not correct

### ISSUE THREAD

**Issue by gabkov** [coinconvert/crypto-convert#6]
https://api.coinconvert.net/convert/ETH/DOGE?amount=10

`{"status":"success","ETH":10,"DOGE":5756497.6265451}`

Correct conversion:

10 ETH = 170,054.26 DOGE

**Comment by el1s7** [comment]
Feel free to open a pull request or try disabling some exchanges
```javascript
convert.set({
	binance: true, //Use binance rates
	bitfinex: true, //Use bitfinex rates
	okex: true, //Use okex rates
});
```

**Comment by el1s7** [comment]
Fixed on v1.1
